mock-services/fake-service/service.js

```diff
@@ -0,0 +1,113 @@
+// Copyright 2017 Istio Authors
```

Comment:
See how to [make this work with RabbitMQ here](https://github.com/Brown-CS2952-cwu-amahajan/baseline-qbox-cluster/pull/1/files#diff-64f0e241fe6d385b1dcb8a7cff01c43a). 

tl;dr add a different file (in PR above, it's called `rabbitmq.js`) containing code for a RabbitMQ consumer and run it as a [forked child process](https://nodejs.org/api/child_process.html#child_process_child_process_fork_modulepath_args_options).

Reply:
I will be taking care of this, so resolving convo.